public/src/main/java/com/rackspace/salus/telemetry/api/web/ResourcesController.java

```diff
@@ -66,11 +60,10 @@ public ResourcesController(UserService userService, ServicesProperties servicesP
     return proxy.uri(backendUri).get();
   }
 
-  @GetMapping("/resourcesByLabel")
+  @GetMapping("/tenant/{tenantId}/resourcesByLabel")
```

Comment:
Perhaps while you're in here
```suggestion
  @GetMapping("/tenant/{tenantId}/resources-by-label")
```

Reply:
Thanks.  I spotted that when I was first looking at the files then forgot to change it.  I'll update the other service's endpoint too.